front/src/applications/stdcm/components/StdcmResults/StdcmFeedback.tsx

```diff
@@ -0,0 +1,73 @@
+import { Button } from '@osrd-project/ui-core';
+import { Comment } from '@osrd-project/ui-icons';
+import dayjs from 'dayjs';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+import type { StdcmResultsOutput } from 'applications/stdcm/types';
+import { getSelectedSimulation } from 'reducers/osrdconf/stdcmConf/selectors';
+import { dateTimeFormatting } from 'utils/date';
+
+const FeedbackCard = () => {
```

Comment:
Just to keep the same name used for the file
```suggestion
const StdcmFeedback = () => {
```

Reply:
done